frontend/scripts/actions/home.actions.js

```diff
@@ -0,0 +1,22 @@
+import actions from 'actions';
+
+import {
+  getURLFromParams,
+  GET_TWEETS,
+  GET_POSTS
+} from 'utils/getURLFromParams';
+
+export const getSliderContent = type => (dispatch) => {
+  const params = {
+    tweets: GET_TWEETS,
+    posts: GET_POSTS
+  }[type];
+  const url = getURLFromParams(params);
+  fetch(url)
+    .then(res => (res.ok ? res.json() : Promise.reject(res.statusText)))
+    .then(data => dispatch({
+      type: actions.SET_SLIDER_CONTENT,
```

Comment:
I would suggest moving away from a single actions file, and having action names defined in this file itself - otherwise we'll end up with a huge actions file, which are hard to tell appart. See GFishW for examples